outputs.tf

```diff
@@ -34,7 +34,7 @@ output "dns_zone_id" {
 
 output "zone_rt_id" {
   description = "A map containing the route table id per availability zone"
-  value       = "${zipmap(aws_subnet.nat_subnets.*.availability_zone, aws_route_table.az_rts.*.id)}"
+  value       = "${zipmap(var.zones, aws_route_table.az_rts.*.id)}"
```

Comment:
did you have to be careful about this?? ... as it might accidentally have cascading effects

Reply:
I believe that's what you'd get from the NAT subnet output anyways? I checked this change against our test cluster and there was no change in the format of the output or id correlation.